templates/base.html

```diff
@@ -24,7 +24,7 @@
                 <li id="anb-forums"><a href="https://bbs.archlinux.org/" title="Community forums">Forums</a></li>
                 <li id="anb-wiki"><a href="https://wiki.archlinux.org/" title="Community documentation">Wiki</a></li>
                 <li id="anb-bugs"><a href="https://bugs.archlinux.org/" title="Report and track bugs">Bugs</a></li>
-                <li id="anb-security"><a href="https://security.archlinux.org/" title="Arch Linux Security Tracker">Security</a></li>
+                <li id="anb-security"><a href="/security/" title="Arch Linux Security">Security</a></li>
```

Comment:
Hmm I'm not sure if that's what we want. We still want people to find the tracker, but also find security contact information.

Reply:
I think  prefacing it with the team and then provide a link to the tracker is good enough in this case.

Reply:
as said on the IRC im very much against this.
We can surely make the contact information page as "home" for the security tracker (if thats what you guys want), but the link should bring you to the security sub-space. that's where it belongs and info should be maintained.